### PR TITLE
Sort struct properties in alphabetic order

### PR DESCRIPTION
**Pull request by kolya182**
For the sake of some consistency sort struct fields in alphabetic order

### CODE REVIEW

File: pkg/blockatlas/tx.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ type (
 	Block struct {
 		Number int64  `json:"number"`
 		ID     string `json:"id,omitempty"`
-		Txs    []Tx   `json:"txs"`
+		Txs    TxPage `json:"txs"`
```
Comment (hewigovens) [inline pkg/blockatlas/tx.go:69]:
```suggestion
		Txs    []Tx `json:"txs"`
```

Comment (kolya182) [reply pkg/blockatlas/tx.go:69]: Sorted via https://codebeautify.org/sort-text-lines, but this change let me fix that

**Comment by EnoRage** [comment]
Goland also have this feature

**Comment by vikmeup** [comment]
Can we sort this for the whole project and add this to linting CI? otherwise it's pointless to make such changes as as make git history quite hard to look after

**Comment by EnoRage** [comment]
@vikmeup Good idea, have plans for it

**Comment by kolya182** [comment]
> @vikmeup Good idea, have plans for it

Issue to track https://github.com/trustwallet/blockatlas/issues/1089

**Pull request closed without merging**